platform/linux-generic/odp_packet_io.c

```diff
@@ -33,9 +33,9 @@
 #include <time.h>
 
 /* Sleep this many nanoseconds between pktin receive calls */
-#define SLEEP_NSEC  1000
+#define SLEEP_USEC  1
```

Comment:
Comment still says nanoseconds. Should also state that the maximum value is 1 000 000, so that timespec.tv_nsec is used correctly.

Reply:
Will fix both problems.